src/rpc/blockchain.cpp

```diff
@@ -757,7 +777,15 @@ UniValue getblock(const UniValue& params, bool fHelp)
             "          \"valueDelta\": xxxxxx,      (numeric, optional) change to the amount in the pool produced by this block, in " + CURRENCY_UNIT + "\n"
             "          \"valueDeltaZat\": xxxxxx,   (numeric, optional) change to the amount in the pool produced by this block, in " + MINOR_CURRENCY_UNIT + "\n"
             "      }, ...\n"
-            "  ]\n"
+            "  ],\n"
+            "  \"trees\": {                 (object) information about the note commitment trees\n"
+            "      \"sapling\": {             (object, optional)\n"
+            "          \"size\": n,             (numeric) the number of note commitments appended to the tree\n"
```

Comment:
```suggestion
            "          \"size\": n,             (numeric) the number of note commitments appended to the tree by this block\n"
```

Reply:
This is not correct; this is the total size of the note commitment tree as of the end of the block. I'll clarify.